.github/workflows/e2e-ci.yml

```
@@ -43,6 +43,9 @@ jobs:
       - name: Make jest available
         working-directory: ${{env.working-directory}}
         run: cp ../../node_modules/.bin/jest node_modules/.bin/
+      - name: Check E2E wallet balance
+        working-directory: ${{env.working-directory}}
+        run: yarn ts-node ./e2e/scripts/check-e2e-wallet-balance.ts
```

We need to fail the workflow in case of an unhandled rejected promise. One way to do it with [ts-node](https://typestrong.org/ts-node/docs/configuration/#node-flags):
```suggestion
        run: NODE_OPTIONS='--unhandled-rejections=strict' yarn ts-node ./e2e/scripts/check-e2e-wallet-balance.ts
```

---

Fixed in https://github.com/valora-inc/wallet/pull/2325/commits/336783cd5921d1d51e9f43703d5fd3acd2a89456!